### PR TITLE
(DOCS-9610) GitHub triggers for workflows

### PR DESCRIPTION
**Pull request by neko-dd**
### What does this PR do? What is the motivation?

Adds documentation for new trigger method for workflows.

### Merge instructions

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### CODE REVIEW

File: content/en/service_management/workflows/trigger.md [hunk 1]
```diff
@@ -152,6 +152,24 @@ You can manually start a workflow from a Cloud SIEM Security Signal panel.
 
 For additional examples of security workflows you can automate, see [Automate Security Workflows with Workflow Automation][4].
 
+## GitHub triggers
+
+<div class="alert alert-info"><strong>Note</strong>: Your GitHub account must have permission to create webhooks for a GitHub repo to use this feature.</div>
```
Comment (GabeWM) [inline content/en/service_management/workflows/trigger.md:157]: Technically GitHub webhooks can be triggered from more than just a repositroy. See [here](https://docs.github.com/en/webhooks/types-of-webhooks) for the different webhook types.  Regardless it is a good callout that you must have to be the owner or admin privileges to create a Github webhook (depends on the webhook type). Not sure if it is worth being explicit here.

Comment (neko-dd) [reply content/en/service_management/workflows/trigger.md:157]: Okay, I can take out the bit about repo

File: content/en/service_management/workflows/trigger.md [hunk 2]
```diff
@@ -152,6 +152,24 @@ You can manually start a workflow from a Cloud SIEM Security Signal panel.
...
+
+You can trigger a workflow from GitHub using the following steps.
+
+1. Add a GitHub trigger to your workflow:
+   - If your workflow doesn't have any triggers, click **Add Trigger > GitHub**.
+   - If your workflow already has one or more triggers and you're adding GitHub as an additional trigger, click the **Add Trigger** (lightning bolt) icon and select **GitHub**.
+1. Navigate to the GitHub repo you want to use to trigger your workflow.
+1. In GitHub, click **Settings**, click **Webhooks**, and then click **New Webhook**.
```
Comment (GabeWM) [inline content/en/service_management/workflows/trigger.md:165]: nit: I see add webhook 

Screenshot 2024-11-26 at 6 28 18 PM

**Review by GabeWM — APPROVED** [review]

**Review by hestonhoffman — APPROVED** [review]
LGTM!

**Pull request merged**